pages/linux/pr.md

```diff
@@ -0,0 +1,11 @@
+# pr
+
+> Format text files for printing.
+
+- Paginate the file for printing:
+
+`pr {{filename}} --pages={{FIRST_PAGE}}[:{{LAST_PAGE}}]`
```

Comment:
Is the unspecified last page syntax that common? I'd suggest removing the `[]` markup to make the example easier to parse.

Note that tldr pages don't need to be exhaustive and demonstrate all usage patterns of a command. But if you feel it's important to demonstrate the omitted last page behavior, we can split the example in two, since the page is still quite short :)